pleskdistup/actions/grub.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023-2024. WebPros International GmbH. All rights reserved.
+import os.path
+
+from pleskdistup.common import action, dist, util
+
+DEBCONF_CMD = "/usr/bin/debconf-show"
+
+class AssertGrubInstallDeviceExists(action.CheckAction):
+    def __init__(self) -> None:
+        self.name = "check GRUB installation device exists"
+        self.description = """Grub's install-device is not found: {}
+This could fail conversion - please fix it.
+"""
+
+    def _do_check(self) -> bool:
+        if not dist.get_distro().deb_based:
+            return True
+
+        debconf_out = util.logged_check_call([DEBCONF_CMD, "grub-pc"])
+        prefix = "grub-pc/install_devices:"
```

Comment:
NIT: We could turn this into a class member and make it configurable through the init function argument. However, it does not seem critical at the moment.

Reply:
Yes, it seems like (to stay) very debian-specific class anyway.